core/pom.xml

```diff
@@ -153,8 +143,12 @@
 			<artifactId>opennlp-tools</artifactId>
 		</dependency>
 		<dependency>
+			<!-- TODO: NemexWekaClassificationEDATest throws "java.lang.Exception: liblinear classes not in CLASSPATH!".
+            This requires a wrapper but the one in https://github.com/bwaldvogel/liblinear-weka is outdated. -->
 			<groupId>de.bwaldvogel</groupId>
 			<artifactId>liblinear</artifactId>
+			<version>2.11</version>
```

Comment:
I see, tricky one! I could solve it with ClassLoaders, by loading one version of liblinear for the weka-based code and another for the rest of eop. But actually I see that the new version of weka is compatible with the new package structure of liblinear. The only problem is you also need a wrapper-jar, which is not available in a public maven repo (as far as I can find). You get it if you use the weka package-manager to install liblinear locally.
Maybe the least-ugly solution for now would be to provide the liblinear library + wrapper as part of the EOP installation.